h/static/scripts/config/identity.js

```diff
@@ -0,0 +1,139 @@
+var JWT_BEARER = 'urn:ietf:params:oauth:grant-type:jwt-bearer';
+
+var _authPromise = null;
```

Comment:
Given this is a browserify module, there's no particular need to give these "private" names.